modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -176,6 +176,14 @@ endif::[]
 For more information, see xref:client-configuration:registration-overview.adoc[].
 
 
+[WARNING]
+====
+Some clients experience SSL errors when you register them in the {productname} {webui} using a bootstrap script.
+This can make some operations unusable, including package management and bootstrapping.
```

Comment:
Well, if you can't bootstrap, for sure you can't do this, right? :-D

If would rephrase this to something like:

> CentOS6 clients can not be registered or managed correctly unless {productname} server is configured to support old encryptions. Check ```Support for old clients``` at configuration:tshoot-clients.adoc[].